src/components/LicenseAmendmentList/LicenseAmendmentList.css

```diff
@@ -1,3 +1,9 @@
+@import "@folio/stripes-components/lib/variables";
+
 .note {
```

Comment:
Looks like you're not using this set anymore so should delete it.

Reply:
Which set is this sorry? The note set I didn't change anything, and the import of variables is used below for var(--warn), so I'm a little confused.

Reply:
I couldn't figure out how to use the `.note` styling to affect <div>, so I stuck to the explicit column width method. I've deleted that `.note` bit now, let me know if that's not what you meant